src/main/java/mate/academy/lib/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.lib;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "Dimension3306");
```

Comment:
Hardcoding passwords in the source code is a security risk. Consider using environment variables or a configuration file to store sensitive information securely.